driver/ppm_fillers.c

```diff
@@ -6730,10 +6730,10 @@ int f_sys_bpf_x(struct event_filler_arguments *args)
 	res = val_to_ring(args, fd, 0, false, 0);
 	CHECK_RES(res);
 
-	/* Parameter 2: cmd (type: PT_INT64) */
+	/* Parameter 2: cmd (type: PT_ENUMFLAGS32) */
 	syscall_get_arguments_deprecated(args, 0, 1, &val);
 	cmd = (int32_t)val;
-	res = val_to_ring(args, cmd, 0, false, 0);
+	res = val_to_ring(args, (uint32_t)cmd, 0, false, 0);
```

Comment:
I'd still leave a `bpf_flags_to_scap()` helper function even if it won't do anything (ie: just `return cmd`, like we do for example for `fsconfig_cmds_to_scap`). It make it more evident that there is a translation going on.

Reply:
Would a comment do instead? Not a fan of jumping unnecessarily, but agree to make it clearer.

Reply:
Given the helpers are inlined, i am pretty sure the compiler is smart enough to drop the jump for us (considering the function would be `int foo(int x) { return x; }`).

Reply:
If you feel very strongly we need it, I won't block it and re-approve. Just wanted to leave a note here that I really don't think it's necessary, a comment will just do.